.gitignore

```diff
@@ -0,0 +1,8 @@
+._*
+.DS_Store
+
+**/.terraform/*
+*.tfstate
+*.tfstate.*
+
```

Comment:
add `*.terraform.lock.hcl`. If you're not checking in state, don't check in the lock file either.

Reply:
whoops, nice catch. thanks!